### ISSUE TITLE
PHP-style code wrapping

### ISSUE THREAD

**Issue by as-com** [PyHP-pph/PyHP-pph#21]
In PHP, only code wrapped within `<?php ... ?>` will be run; the stuff outside is printed verbatim. PyHP++# should have similar behavior.

**Comment by boynedmaster** [comment]
Agreed. This will also make the PyHP++# "Hello, world!" very simple, making this masterpiece of a language more inviting to newcomers.

**Comment by ghost** [comment]
Plus, it means that any program that doesn't contain a `<?pyph++# ... ?>` (why not use the actual symbols?) tag is automatically a quine!
